### ISSUE TITLE
Rebuild database with new data set

### ISSUE THREAD

**Issue by russellgould** [hackoregon/emergency-response#37]
The new data we got is (I think, maybe) everything from 6am Jan 1st, 2010, through 1am Jan 1st, 2017. As such we don't need the previous data set we have in our database. Additionally, this new data has some extra columns in (at least one of) the important tables, so we will probably just get rid of what we previously had and replace it with the new hotness.

**Comment by sky-t** [comment]
Tables are uploaded to AWS instance but still need key constraints added.

**Comment by sky-t** [comment]
This action is now complete. Full dataset is now on the AWS instance. However, there is one minor (I beleive) caveat. We were not able to implement this fkey constraint in the responder table ```FOREIGN KEY (Incident_ID) REFERENCES incident (Incident_ID)```  due to this error:
```ERROR:  insert or update on table "responder" violates foreign key constraint "responder_incident_id_fkey"
DETAIL:  Key (incident_id)=(754223) is not present in table "incident".
```
Looking at the inctimes table where incident_id 754223 also has row entries,  I believe the reason that this incident_id is missing from the incidents table is due to the fact that it starts on 2009-12-31. If we can get the missing incident table row for this incident_id from @markawhitaker we should be able to re-implement the fkey constraint on the responder table. If re-implementing this foreign key is deemed non necessary, we can close this action item.

**Comment by russellgould** [comment]
Blocked by #46

**Comment by russellgould** [comment]
Fixed #46 and will consider this now done.
